pkgs/by-name/ba/base91/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "base91";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "douzebis";
+    repo = "base91";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Lx569OtNU0z884763FSByH0yXIq6GzeXyp5NVvMejco="; # update as needed
+  };
+
+  makeFlags = [
+    "CC=gcc"
```

```suggestion
    "CC=cc"
```

`cc` is an alias both `clang` and `gcc` use.

---

Thank you.